src/main/java/com/gengo/client/GengoClient.java

```diff
@@ -135,12 +126,12 @@ public JSONObject postTranslationJobs(List<TranslationJob> jobs, boolean process
             JSONObject rsp = call(url, HttpMethod.POST, data);
             return rsp;
         }
-        catch (JSONException x)
+        catch (JSONException e)
         {
             throw new GengoException(x.getMessage(), x);
```

Comment:
Oh, I'm so sorry. this is a terrible mistake...
I'll update the line and same mistakes. Thank you very much.